src/universal-dapp.js

```diff
@@ -562,24 +556,7 @@ UniversalDApp.prototype.runTx = function( data, args, cb) {
         }
     }
 
-    if (!this.vm) {
```

Comment:
Can you please have this part of the change as a separate commit? It doesn't seem to be related, but rather cosmetical.